services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> list:
```

Comment:
it's better to use Django's queryset type hint QuerySet instead of a generic list.